scipy/linalg/lapack.py

```diff
@@ -482,7 +482,7 @@ def get_lapack_funcs(names, arrays=(), dtype=None):
         used, otherwise the most generic type in arrays will be used.
 
     dtype : str or dtype, optional
-        Data-type specifier. Not used if `arrays` is non-empty.
+        Data-type specifier. Not used if ``arrays`` is non-empty.
```

Comment:
In theory these are supposed to link although I've never seen it happen. Everywhere else in SciPy we use single-backticks for variables / input names.